_posts/en/newsletters/2025-04-04-newsletter.md

```diff
@@ -464,7 +464,95 @@ Bitcoin's consensus rules._
+  being spent to be transferred to the output being created---commonly
+  (but [controversially][towns anticov]) called a [covenant][topic
+  covenants] in Bitcoin jargon.  The covenant may allow satisfaction or
+  modification of the encumbrance, which would (respectively) terminate
+  the covenant or modify its terms for future iterations.  Ingala
+  describes some of the benefits and drawbacks of this approach:
+
+  - *Benefits:* taproot native, doesn't increase the size of taproot
+    entries in the UTXO set, and spending paths that don't require the
+    extra data don't need to include it in their witness stack (so
+    there's no extra cost in that case).
+
+  - *Drawbacks:* only works with taproot and checking tweaks requires
+    elliptical curve operations that are more expensive than (say)
+    SHA256 operations.
+
+  Only transferring the spending conditions from the output being spent
+  to an output being can be useful, but many covenants will want to
+  ensure some or all of the bitcoins in the output being spent are
+  passed through to the output being created.  Ingala describes CCV's
+  three options for handling values.
+
+  - *Ignore:* don't check amounts.
+
+  - *Deduct:* the amount of an output being created at a particular
+    index (e.g., the third output) is deducted from the amount of the
+    output being spent at the same index and the residual value is
+    tracked.  For example, if the output being spent at index three is
+    worth 100 BTC and the output being created at index three is worth
+    70 BTC, then the code keeps track of the residual 30 BTC.  The
+    transaction is marked as invalid if the output being created is
+    greater than the output being spent (as that would reduce the
+    residual value, perhaps below zero).
+
+  - *Default:* mark the transaction invalid unless the amount of output
+    being created at a particular index is greater than the amount of
+    the output being spent plus the sum of any previous residuals that
+    haven't been used with a _default_ check yet.
+
+  A transaction is valid if any output is checked more than once with
+  _deduct_ or if both _deduct_ and _default_ are used on the same
+  output.
+
+  Ingala provides several visual examples of combinations of the above
+  operations.  Here's our textual description of his "send partial
+  amount" example, which might be useful for a [vault][topic vaults]: a
+  transaction has one input (spending one output) worth 100 BTC and two
+  outputs, one for 70 BTC and the other for 30 BTC.  CCV is run twice
+  during validation of the input:
+
+  1. CCV with _deduct_ operates index 0 for 100 BTC spent to 70 BTC
+     created, giving a residual of 30.  In a [BIP345][]-style vault, CCV
+     would be returning that 70 BTC back to the same script it was
+     previously secured by.
+
+  2. The second time, it uses _default_ on index 1.  Although there's an
+     output being created at index 1 in this transaction, there's no
```

Comment:
```suggestion
  2. The second time, it uses _default_ on index 1.  Although an
     output is created at index 1 in this transaction, there's no
```

Reply:
Leaving as-is so that we use the same "output being created" term here as we use in the rest of the section.